### DISCUSSION TITLE
FORGOT MY PASSOWORD ON FIREFLY WHAT CAN I DO ?

### DISCUSSION THREAD

**Discussion by Happyponytale** [iotaledger/firefly#5202] [Help & Support]
Can anyone help?

**Comment by HankMoody089** [comment]
Just setup a new profile with your 24 words or Ledger or Stronghold Backup.

**Reply by Happyponytale** [reply]
thanks man x

**Comment by HankMoody089** [comment]
Next time come to IOTA Discord for asking questions, please.
